src/main/java/com/careerfair/q/controller/employee/implementation/EmployeeControllerImpl.java

```diff
@@ -41,4 +47,10 @@ public DeleteEmployeeResponse deleteEmployee(@PathVariable("id") String id) {
     public AddEmployeeResponse addEmployee(@RequestBody AddEmployeeRequest addEmployeeRequest) {
         return employeeService.addEmployee(addEmployeeRequest);
     }
+
+    @GetMapping("/get/fair/")
+    @Override
+    public GetAllFairsResponse getAllFairs() throws ExecutionException, InterruptedException {
```

Comment:
Should this method be here? Maybe another service? or aren't we assuming that there will only be one career fair?